core/arch/arm/plat-versal/conf.mk

```diff
@@ -0,0 +1,31 @@
+PLATFORM_FLAVOR ?= generic
+
+include core/arch/arm/cpu/cortex-armv8-0.mk
+
+$(call force,CFG_TEE_CORE_NB_CORE,2)
+$(call force,CFG_PL011,y)
+$(call force,CFG_GIC,y)
+$(call force,CFG_ARM_GICV3,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+
+# Disable core ASLR for two reasons:
+# 1. There is no source for ALSR seed, as ATF does not provide a
+#    DTB to OP-TEE. Hardware RNG is also not currently supported.
+# 2. OP-TEE does not boot with enabled CFG_CORE_ASLR.
+$(call force,CFG_CORE_ASLR,n)
+
+ifeq ($(CFG_ARM64_core),y)
+$(call force,CFG_CORE_ARM64_PA_BITS,43)
+$(call force,CFG_WITH_LPAE,y)
+else
+$(call force,CFG_ARM32_core,y)
+endif
+
+CFG_TZDRAM_START ?= 0x60000000
+CFG_TZDRAM_SIZE  ?= 0x10000000
+CFG_SHMEM_START  ?= 0x70000000
+CFG_SHMEM_SIZE   ?= 0x10000000
+
+CFG_WITH_STATS ?= y
+CFG_CRYPTO_WITH_CE ?= n
```

Comment:
Interesting. If CE is supported at all, why not enable it by default?

Reply:
my bad, just reused it from zynqmp. will remove as it doesnt seem to be supported.

Reply:
Then `$(call force,CFG_CRYPTO_WITH_CE,n)` would be appropriate. But that seems weird, 64-bit core without crypto extensions?

Reply:
my mistake (again!) - I did enable support and assumed that the crash I was seeing was due to that and didnt think twice.
CEs are not imposed by the architecture so even though optional I agree it would be weird not to have them.

Just checked the TRM and it is indeed supported. So yes, will force it 

Versal ACAP
Technical Reference Manual
AM011 (v1.3) October 27, 2021

```
Application Processing Unit
The APU MPCore processor is integrated into the FPD subsystem. The implementation and
functionality of the Cortex®-A72 processor is detailed in the Application Processing Unit chapter
with additional information in the Arm documents.
The main features of the APU:
• Dual Arm Cortex-A72 cores
• VFPv4 floating point, NEON, Crypto extension
• 48 KB instruction, 32 KB data caches
• GIC-500 interrupt controller
• 1 MB L2 Cache Coherent Interconnect (CCI)
```

Reply:
`CFG_CRYPTO_WITH_CE ?= y` would be slightly more correct (allowing the developer to disable it at build time), but it doesn't really matter.

Reply:
ok